### PR TITLE
Add reviewers.md

### PR DESCRIPTION
**Pull request by rshym**

### CODE REVIEW

**Comment by lorc** [comment]
`Acked-by:Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

We will need formal Acked-by from all persons mentioned in the commit.

File: reviewers.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+| android_vendor_xen_audio        | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_vendor_xen_gnss         | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_vendor_xen_sensors      | Andrii Chepurnyi     | Ruslan Shymkevych |
+| android_vendor_xen_vehicle      | Andrii Chepurnyi     | Ruslan Shymkevych |
+| camera_be                       | Andrii Chepurnyi     | Oleksandr Grytsov |
+| DisplayManager                  | Andrii Chepurnyi     | Oleksandr Grytsov |
+| displ_be                        | Andrii Chepurnyi     | Oleksandr Grytsov |
+| kmgk                            | Andrii Chepurnyi     | Ruslan Shymkevych |
+| libvisclient                    | Andrii Chepurnyi     |                   |
+| libxenbe                        | Andrii Chepurnyi     |                   |
```
Comment (al1img) [inline reviewers.md]: Please add me as libxenbe as well.

**Comment by arminn** [comment]
Acked-by: Andrii Chepurnyi <andrii_chepurnyi@epam.com>

File: reviewers.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+| moulin-rcar-gen3                | Volodymyr Babchuk    |                   |
+| optee_os                        | Volodymyr Babchuk    | Andrii Chepurnyi  |
+| qemu                            |                      |                   |
+| snd_be                          | Oleksandr Grytsov    |                   |
```
Comment (al1img) [inline reviewers.md]: Main reviewer should be @andr2000,  I will be second one.

File: reviewers.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+| xt-log                          | Oleksandr Grytsov    |                   |
+
+## Previous generation projects
+| Repo                      | Reviewer          | Reviewer |
+|---------------------------|-------------------|----------|
+| build-scripts             | Ruslan Shymkevych |          |
+| manifests                 | Ruslan Shymkevych |          |
+| meta-xt-agl-base          | Ruslan Shymkevych |          |
+| meta-xt-images            | Ruslan Shymkevych |          |
+| meta-xt-prod-aos          | Ruslan Shymkevych |          |
```
Comment (al1img) [inline reviewers.md]: Put me here as well. Also, we close support of this product. I mean new features will not be added there. We have
created new product based on moulin: https://github.com/xen-troops/meta-xt-prod-aos-rcar. Please add it. If you would like you can be reviewer at this repo as well. But also add me.

File: reviewers.md (hunk 1 above)
Comment (rshym) [reply reviewers.md]: Added.

File: reviewers.md (hunk 3 above)
Comment (rshym) [reply reviewers.md]: Added as reviewer to meta-xt-prod-aos.
Repo meta-xt-prod-aos-rcar is added with you as reviewer.

**Comment by rshym** [comment]
- meta-xt-prod-aos-rcar is added
- reviewer is added to meta-xt-prod-aos
- reviewer is added to sndbe and xenbe

**Review by al1img — COMMENTED** [review]
`Acked-by: Oleksandr Grytsov <oleksandr_grytsov@epam.com>`

**Comment by otyshchenko1** [comment]
> Acked-by: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>

**Pull request merged**